packages/muelu/src/MueCentral/MueLu_MasterList.cpp

```diff
@@ -161,7 +161,7 @@ namespace MueLu {
   "<Parameter name=\"smoother: pre overlap\" type=\"int\" value=\"0\"/>"
   "<Parameter name=\"smoother: post overlap\" type=\"int\" value=\"0\"/>"
   "<Parameter name=\"coarse: max size\" type=\"int\" value=\"2000\"/>"
-  "<Parameter name=\"coarse: type\" type=\"string\" value=\"SuperLU\"/>"
+  "<Parameter name=\"coarse: type\" type=\"string\" value=\"KLU\"/>"
```

Comment:
This should not be changed here, but instead at at https://github.com/trilinos/Trilinos/blob/436ce065b6e3c37acc3f5db6807a9cdf9807c78f/packages/muelu/doc/UsersGuide/masterList.xml#L356, then run `update_params.sh`.

Reply:
There is also a small chance this will require gold file changes in some downstream packages.

Reply:
I'll wait and see what the autotester says. Thanks!